nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/createindex/CreateIndexServiceImpl.java

```diff
@@ -93,8 +98,23 @@ private void parseAssetIntoChartEntry(final ChartIndex index, final Asset asset)
     chartEntry.setDescription(formatAttributes.get(DESCRIPTION.getPropertyName(), String.class));
     chartEntry.setIcon(formatAttributes.get(ICON.getPropertyName(), String.class));
     chartEntry.setCreated(asset.blobCreated());
+
+    @SuppressWarnings("unchecked")
+    List<String> maintainers = formatAttributes.get(MAINTAINERS.getPropertyName(), List.class);
+    if (!CollectionUtils.isEmpty(maintainers)) {
+      List<Map<String, String>> nextMaintainers = maintainers
+          .stream()
+          .map(param -> {
+            String[] split = param.split(HelmAttributes.MAINTAINERS_SEPARATOR);
+            String key = split[0];
+            String value = split[1];
+            return Collections.singletonMap(key, value);
+          })
+          .collect(Collectors.toList());
+      chartEntry.setMaintainers(nextMaintainers);
+    }
+
```

Comment:
move this block to a separate function

Reply:
Fixed. 
https://github.com/sonatype-nexus-community/nexus-repository-helm/pull/92/commits/fcb1480ed325222a09c2cd0124cd92f67e7bf676